firefox-ios/Client/Frontend/Library/LibraryViewController/LibraryViewController.swift

```diff
@@ -325,6 +326,19 @@ class LibraryViewController: UIViewController, Themeable {
         navigationController?.toolbar.tintColor = theme.colors.actionPrimary
     }
 
+    private func updateSegmentControl() {
+        if isBookmarkRefactorEnabled {
```

Comment:
nit, I would use `guard isBookmarkRefactorEnabled else { return }` here instead of an `if` to reduce nesting.
```suggestion
        if isBookmarkRefactorEnabled {
```